Detour/Source/DetourNavMeshQuery.cpp

```diff
@@ -481,9 +481,9 @@ dtStatus dtNavMeshQuery::findRandomPointAroundCircle(dtPolyRef startRef, const f
 	dtRandomPointInConvexPoly(verts, randomPoly->vertCount, areas, s, t, pt);
 	
 	float h = 0.0f;
-	dtStatus stat = getPolyHeight(randomPolyRef, pt, &h);
+	status = getPolyHeight(randomPolyRef, pt, &h);
```

Comment:
You are right. I will write it the same way I did for `findPath`.
Currently on vacation though, will be back in a couple of weeks.